File: data/recomposed_projects/llama-3-70b-instruct/body/commons-graph/evosuite-test/src/test/org/apache/commons/graph/utils/Objects_ESTest.py
from __future__ import annotations
import time
import re
import os
import typing
from typing import *
import unittest
import pytest
import io
import unittest
from src.main.org.apache.commons.graph.utils.Objects import *

# from src.test.org.apache.commons.graph.utils.Objects_ESTest_scaffolding import *
# from src.main.org.evosuite.runtime.EvoAssertions import *
# from src.main.org.evosuite.runtime.EvoRunnerParameters import *


class Objects_ESTest(unittest.TestCase):

    def test6(self) -> None:
        boolean0 = Objects.eq(None, None)
        self.assertTrue(boolean0)

    def test5(self) -> None:

        integer0 = Integer(1)
        object0 = Object()
        boolean0 = Objects.eq(integer0, object0)
        self.assertFalse(boolean0)

    def test4(self) -> None:

        integer0 = Integer(-589)
        boolean0 = Objects.eq(None, integer0)
        self.assertFalse(boolean0)

    def test3(self) -> None:

        objectArray0 = []
        int0 = Objects.hash(1, 1, objectArray0)
        self.assertEqual(1, int0)

    def test2(self) -> None:

        object0 = object()
        objectArray0 = [None] * 5
        objectArray0[0] = object0
        int0 = Objects.hash(0, 0, objectArray0)
        self.assertEqual(0, int0)

    def test1(self) -> None:

        try:
            Objects.hash(0, 0, None)
            self.fail("Expecting exception: RuntimeError")

        except TypeError as e:
            self.verifyException("org.apache.commons.graph.utils.Objects", e)

    def test0(self) -> None:

        object0 = object()
        objectArray0 = [None] * 5
        objectArray0[0] = object0
        int0 = Objects.hash(-419, 1335, objectArray0)
        self.assertEqual(-1871240068, int0)

File: java_projects/cleaned_final_projects_evosuite/commons-graph/evosuite-tests/org/apache/commons/graph/utils/Objects_ESTest.java

/*
  Licensed to the Apache Software Foundation (ASF) under one or more
  contributor license agreements.  See the NOTICE file distributed with
  this work for additional information regarding copyright ownership.
  The ASF licenses this file to You under the Apache License, Version 2.0
  (the "License"); you may not use this file except in compliance with
  the License.  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
 */


package org.apache.commons.graph.utils;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.apache.commons.graph.utils.Objects;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true) 
public class Objects_ESTest extends Objects_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Object object0 = new Object();
      Object[] objectArray0 = new Object[5];
      objectArray0[0] = object0;
      int int0 = Objects.hash((-419), 1335, objectArray0);
      assertEquals((-1871240068), int0);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      // Undeclared exception!
      try { 
        Objects.hash(0, 0, (Object[]) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("org.apache.commons.graph.utils.Objects", e);
      }
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      Object object0 = new Object();
      Object[] objectArray0 = new Object[5];
      objectArray0[0] = object0;
      int int0 = Objects.hash(0, 0, objectArray0);
      assertEquals(0, int0);
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      Object[] objectArray0 = new Object[0];
      int int0 = Objects.hash(1, 1, objectArray0);
      assertEquals(1, int0);
  }

  @Test(timeout = 4000)
  public void test4()  throws Throwable  {
      Integer integer0 = new Integer((-589));
      boolean boolean0 = Objects.eq((Integer) null, integer0);
      assertFalse(boolean0);
  }

  @Test(timeout = 4000)
  public void test5()  throws Throwable  {
      Integer integer0 = new Integer(1);
      Object object0 = new Object();
      boolean boolean0 = Objects.eq((Object) integer0, object0);
      assertFalse(boolean0);
  }

  @Test(timeout = 4000)
  public void test6()  throws Throwable  {
      boolean boolean0 = Objects.eq((Integer) null, (Integer) null);
      assertTrue(boolean0);
  }
}
